src/lib.rs

```diff
@@ -1448,7 +1439,7 @@ mod tests {
             let unused_key = find_unused_environment_variable();
             env::set_var(&unused_key, "foo");
             let StdoutTrimmed(output) =
-                run_output!(test_helper(), "echo", &unused_key, Env(unused_key, "bar"));
+                run_output!(test_helper(), "echo", &unused_key, Env(&unused_key, "bar"));
```

Comment:
This is now needed because `run_output!` constructs a tuple of all input values.